packages/ui-components/src/components/relative-time-picker/relative-time-picker.config.ts

```diff
@@ -0,0 +1,429 @@
+		{
+			label: 'Last 15 mins',
+			value: 'last-15-m',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -15,
+				unit: 'minutes'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'HH:mm',
+				separator: 'to',
+				endDateFormatter: 'HH:mm'
+			}
+		},
+		{
+			label: 'Last 30 mins',
+			value: 'last-30-m',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -30,
+				unit: 'minutes'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'HH:mm',
+				separator: 'to',
+				endDateFormatter: 'HH:mm'
+			}
+		},
+		{
+			label: 'Last 1 hour',
+			value: 'last-1-h',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -1,
+				unit: 'hour'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'HH:mm',
+				separator: 'to',
+				endDateFormatter: 'HH:mm'
+			}
+		},
+		{
+			label: 'Last 2 hours',
+			value: 'last-2-h',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -2,
+				unit: 'hours'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'HH:mm',
+				separator: 'to',
+				endDateFormatter: 'HH:mm'
+			}
+		},
+		{
+			label: 'Last 6 hours',
+			value: 'last-6-h',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -6,
+				unit: 'hours'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'HH:mm',
+				separator: 'to',
+				endDateFormatter: 'HH:mm'
+			}
+		},
+		{
+			label: 'Last 12 hours',
+			value: 'last-12-h',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -12,
+				unit: 'hours'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'HH:mm',
+				separator: 'to',
+				endDateFormatter: 'HH:mm'
+			}
+		},
+		{
+			label: 'Last 24 hours',
+			value: 'last-24-h',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -24,
+				unit: 'hours'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM HH:mm',
+				separator: 'to',
+				endDateFormatter: 'D MMM HH:mm'
+			}
+		},
+		{
+			label: 'Last 48 hours',
+			value: 'last-48-h',
+			comparisonConfig: ERelativeTimeComparisonConfig.RelativeAmountOfUnits,
+			startDate: {
+				amount: -48,
+				unit: 'hours'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM HH:mm',
+				separator: 'to',
+				endDateFormatter: 'D MMM HH:mm'
+			}
+		}
+	],
+	[
+		{
+			label: 'Last week',
+			value: 'last-1-w',
+			comparisonConfig: ERelativeTimeComparisonConfig.AbsoluteAmountOfUnits,
+			startDate: {
+				amount: -1,
+				unit: 'week'
+			},
+			endDate: {
+				amount: -1,
+				unit: 'week'
+			},
+			labelRangeFormatter: {
+				startDateFormatter: 'D MMM',
+				separator: 'to',
+				endDateFormatter: 'D MMM'
+			}
+		},
+		{
+			label: 'Last 7 days',
+			value: 'last-7-d',
```

Comment:
These "values" look like keys to me, probably will be better if we move them to an enum.

Reply:
Dependent on the comment above regarding the config file

Reply:
Will look at this refactor after full implementation 👍